docs/conf.py

```diff
@@ -0,0 +1,89 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# This file only contains a selection of the most common options. For a full
+# list see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+import datetime
+import os
+import sys
+
+import pytz
+
+sys.path.insert(0, os.path.abspath("../src"))
+
+import epitools  # noqa: E402
+
+# -- Project information -----------------------------------------------------
+
+project = "EpiTools"
+year = datetime.datetime.now(tz=pytz.timezone("GMT")).year
+author = "EpiTools Authors"
+copyright = f"2023-{year}, {author}"  # noqa: A001
+
+# The full version, including alpha/beta/rc tags
+release = f"v{epitools.__version__}"
+version = "dev" if "dev" in release else release
```

Comment:
Will be handled by `setuptools-scm`
```suggestion
release = version
```